cord19-explorer/src/main/java/edu/asu/diging/cord19/explorer/core/service/worker/CoordinateCleaner.java

```diff
@@ -0,0 +1,15 @@
+package edu.asu.diging.cord19.explorer.core.service.worker;
+
+import edu.asu.diging.cord19.explorer.core.model.Publication;
+
+public interface CoordinateCleaner {
+
+	void cleanCoordinates(String taskId);
```

Comment:
indent

Reply:
Also not sure what to do about this. All indentation matches on my local.